src/common/sound.cpp

```diff
@@ -14,6 +14,34 @@ Sound::Sound() {
     frequency = 100.f; // frequency of sound signal (0-1000)
     volume = 0.50;     // volume of sound signal (0-1)
 
+    // -- values of sound signals - frequencies, volumes, durations
+    // -- durations of beep
+    durations[eSOUND_TYPE_ButtonEcho] = 100.f;
```

Comment:
I'd recommend doing some static initialization. The compiler might understand your code and optimize it away completely, but I wouldn't rely on it.
`static const float durations[] = {100.F, 500.F ....};`
or even better - make a struct and initialize all components together (will be more readable)
`struct DFV { float duration, freq, volume; }; DFV dfv[] = { { 100.F, 900.F, volume }, { .... } };` if volume is a `static const float volume = 0.5F`

Reply:
Also, when you write comments (which is good), please use the doxygen style and prefer documenting the public interface methods in the header file (their purpose and meaning of parameters).
Implementation-related comments are ok to put near the implementation (i.e. into the cpp file)

Reply:
Doxygen commentary studied and implemented and static defining of signal's values changed.